# ADHD

Reasearchers examined 2 groups of children with ADHD. One group was awarder with extra time at recess, another exchanged points for gifts in school gift shop.

**Independent variable**: type of incentive 

**Dependent variable**: the number of good behaviors in a single 20 minute class period

**Null hypothesis**: there will be no difference in the number of good behaviors between two incentive conditions.

In [3]:
import numpy as np
import pandas as pd

n1 = 10  # recess group
n2 = 10  # prize group
# degrees of freedom
df = n1 + n2 - 2
t_crit = 2.101
mean1 = 10
mean2 = 7
se = 0.94 # standard error

In [4]:
t = (mean1 - mean2) / se
t

3.191489361702128

In [5]:
# we reject null
pooled_sd = 2.33
cohen_d = (mean1 - mean2) / pooled_sd
cohen_d

1.2875536480686696

In [7]:
r2 = t**2/(t**2 + df)
r2

0.3613761202659729

**36%** of the variability in good behavior is due to the different incentive conditions.